cli/cmd/diagnose.go

```diff
@@ -160,6 +234,82 @@ func saveLogsToGzipFileInBatches(logFile *os.File, logStream io.ReadCloser, buff
 	return nil
 }
 
+func fetchOdigosCRDs(ctx context.Context, kubeClient *kube.Client, crdDir string) error {
```

Comment:
nit: 
```suggestion
func fetchOdigosCRs(ctx context.Context, kubeClient *kube.Client, crdDir string) error {
```

We are not fetching the definition, we are fetching the resource itself